docs/howto/operate/new-cluster/aws.md

```diff
@@ -0,0 +1,152 @@
+(new-cluster:aws)=
+# Add a new AWS cluster
+
+We do not currently use Terraform to deploy a new cluster in AWS.
+Instead, we use a tool [called `kops`](https://github.com/kubernetes/kops) to provision the cluster.
```

Comment:
FYI, we also use the `eksctl` tool to deploy EKS-based clusters, but there are not docs about that yet (AFAIK, the only EKS cluster we currently have is the carbonplan one: https://github.com/2i2c-org/pilot-hubs/tree/master/eksctl).

Reply:
I added a quick note about `eksctl` to the page, and noted that we should update the content when we solidify our practices there. That work?